### DISCUSSION TITLE
Make `--locations` option "default" and add `--no-location` instead

### DISCUSSION THREAD

**Discussion by renepupil** [docker/scout-cli#132] [Ideas]
I think it will mitigate "Layer 8 issues" like mine (https://github.com/docker/scout-cli/issues/131)
Obviously I could have checked the CLI, but didn't consider such an option even exists...

**Comment by cdupuis** [comment]
Thanks for the suggestion @renepupil. Pinging our PM @jonnyrobots to discuss this further.
